### ISSUE TITLE
Update cmor_version identifier in all tables

### ISSUE THREAD

**Issue by durack1** [PCMDI/cmip6-cmor-tables#142]
All current tables include [`"cmor_version": "3.2",`](https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_3hr.json#L6), should this be updated to `"cmor_version": "3.3",`?

@dnadeau4 @taylor13

**Comment by taylor13** [comment]
Not sure about this, but I thought the cmor_version was supposed to list the first version of CMOR that would be able to read and interpret a table correctly.  If changes are made in CMOR that don't affect its ability to read a CMOR table there would then be no reason for the entry for cmor_version to be changed in the table.  The rationale for this is that a user should be able to determine which tables can be read by his version of CMOR3; the answer is "any version later than the one listed in the table".  
Is there a better approach?

**Comment by durack1** [comment]
@taylor13 fair enough, it may be useful to rename this `cmor_version_minimum` or similar, which would more accurately convey what the information provides.. If it's too hard, then we can just close the issue

**Comment by taylor13** [comment]
@dnadeau4 is cmor_version read from these tables?  If so, is it used in any way by CMOR3?

If the answer is no to either one of the above, then Paul's suggestion to rename cmor_version *in the tables* as "cmor_version_minimum" (or similar) is probably a good idea.

**Comment by taylor13** [comment]
Perhaps we should include all 3 integers in cmor_version, and it's value should be set to the latest existing version of CMOR (at the time any of the tables was modified).

**Comment by wachsylon** [comment]
From [following lines](https://github.com/PCMDI/cmor/blob/a5ebb5d1dc2d0f810e643ffdd34ea0ac0cfd2daf/Src/cmor.c#L3015) one can see, the cmor_version global attribute is the version number of the program. 

> Perhaps we should include all 3 integers in cmor_version, and it's value should be set to the latest existing version of CMOR (at the time any of the tables was modified).

Nevertheless, the information about the minimum CMOR-version suitable to apply the table (with all 3 integers) is of value. For example, if an update of CMOR to a newer version is not possible but more recent tables could be used.

**Comment by doutriaux1** [comment]
@wachsylon that was how i originally intended this to be used in the tables, if some changes are made to the table that require changes to CMOR, then the attribute would be `upped` to the minimum version of CMOR that can read this table. Now, thinking about it, I am not 100% sure that once CMOR includes the `new` requirement it is still able to read the old tables. @taylor13 should we add something to CMOR to reflect that? Or do we assume that once a table is updated one should never again use the old ones?

**Comment by slavakharin** [comment]
We started using CMOR3 and got confused about this flag as well. Our initial expectation was that the value should have been 3.3.3, the same as the CMOR software version, until we stumbled on this issue. If this is still the case that the flag indicates the minimum version of CMOR capable of reading the tables, I like the idea of renaming it to cmor_version_minimum.
